core/include/vecmem/utils/tuple.hpp

```diff
@@ -95,36 +96,28 @@ struct tuple<T, Ts...> {
     ///
     template <typename U, typename... Us,
               std::enable_if_t<vecmem::details::conjunction<
-                                   std::is_constructible<T, U &&>,
-                                   std::is_constructible<Ts, Us &&>...>::value,
```

Comment:
Same as above.

Reply:
On this one I'm less sure. :frowning: The `tail` variable we definitely take as an rvalue reference. But `head` is still a universal reference. So I'm honestly not sure what the most correct test is here.

With all this said, I don't feel all too strongly about these, so the rvalue references might as well go back.